src/common/core/metrics.py

```diff
@@ -0,0 +1,24 @@
+import prometheus_client
+
+from common.core.utils import get_version_info
+from common.prometheus.utils import with_labels
+
+flagsmith_build_info = prometheus_client.Gauge(
+    "flagsmith_build_info",
+    "Flagsmith version and build information",
+    ["ci_commit_sha", "version"],
+    multiprocess_mode="livemax",
```

Comment:
I've not tested this, but should this use `liveall` instead? This gauge will always have a value of `1`, so I'm not sure how Prometheus decides which value to return.

My reasoning is, if for some reason we ever export inconsistent values or labels in this metric, we'd want to know about it. Maybe I'm overthinking this and it's not a real concern.